File: kitten-house-care/src/main/java/org/jesperancinha/housing/model/Cat.java
package org.jesperancinha.housing.model;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class Cat {
    private Long id;

    private String name;

    private String color;

    private String species;

    private String pattern;

    private Long age;

    private List<Long> formerOwners = new ArrayList<>();

    private List<Long> careCenters = new ArrayList<>();
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/model/Cat.kt
package org.jesperancinha.housing.model

import lombok.Data

@Data
class Cat {
    private val id: Long? = null
    private val name: String? = null
    private val color: String? = null
    private val species: String? = null
    private val pattern: String? = null
    private val age: Long? = null
    private val formerOwners: List<Long> = ArrayList()
    private val careCenters: List<Long> = ArrayList()
}